config/content.json

```diff
@@ -21,7 +21,7 @@
             "/docs/autoscale/v1/developer-guide/": "https://github.com/rackerlabs/otter/",
             "/docs/cloud-servers/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-servers/",
             "/docs/cloud-files/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-files/",
-            "/docs/metrics/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-metrics/",
+            "/docs/metrics/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-metrics/",
```

Comment:
Yes, you can manage redirects, too, by adding [an entry to `config/redirects.json`](https://github.com/rackerlabs/nexus-control/blob/master/config/rewrites.json#L59-L64):

```json
{
   "from": "^\\/docs\\/metrics\\/v1\\/developer-guide\\/?(.*)$",
   "to": "/docs/metrics/v2/developer-guide/$1",
   "rewrite": false,
   "status": 301
},
```

That's:

 * An (escaped) regular expresson matching incoming paths;
 * The destination path pattern, including the capture group for "rest of the path";
 * Whether or not the URL should be rewritten in place or actually redirected through the client's browser;
 * and the [HTTP status code to use for the redirect call](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#3xx_Redirection).

> Because docs.rackspace.com is going to the v1 Uri.

Double redirects are bad for SEO. We should get _those_ redirects updated to the new paths, too, once this is merged. Does @steveortiz handle those?